sonic-sensormond/tests/mocked_libs/sonic_platform/chassis.py

```diff
@@ -0,0 +1,22 @@
+"""
+    Mock implementation of sonic_platform package for unit testing
+"""
+
+# TODO: Clean this up once we no longer need to support Python 2
+import sys
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    import mock
+
+from sonic_platform_base.chassis_base import ChassisBase
+
+
+class Chassis(ChassisBase):
+    def __init__(self):
+        ChassisBase.__init__(self)
+        self._eeprom = mock.MagicMock()
+        self._thermal_manager = mock.MagicMock()
```

Comment:
Same here, let's try to leverage the same file as thermalctld. If duplication is really needed due to lack of support for sharing these, then remove `self._thermal_manager` since it is unused.

Reply:
ack on removing thermal_manager.